### ISSUE TITLE
World scale was too low in original VR implementation

### ISSUE THREAD

**Issue by budwheizzah** [Nibre/MotherVR#1]
I'll be checking this out later. I was on my way to finishing this game with the DK2, but it looks like I should put that old venerable headset down and get with the times! That being said I know the world scale is so far off with the DK2 (by about 40%) that the people appear like 4 foot tall figurines. Everything feels cramped. Do consider having an option or parameter to adjust world scale.

Also, for recentering your view with mouse and keyboard (You should avoid playing this game with the gamepad, you'll get sick) have the gamepad plugged in and nearby and press both triggers to recenter. Return to the mouse & kb for play. Whenever you do this, the mouse stops responding until you hit movement keys again; no big deal.

**Comment by budwheizzah** [comment]
Tested... amazing work so far. Donated 5$, it's worth ten times that. Will consider giving more as you expand this. World scale seems improved, but the biggest "must" right now is getting the mirror image to work somehow. Oculus games can currently only be captured/recorded using OBS's capture desktop feature, as any attempt to capture an Oculus game directly results in a blank image (including OculusMirror.exe). Since the game itself requires being in the foreground for input to work, using desktop capture isn't even possible at the moment.

**Comment by StarLightPL** [comment]
I strongly disagree about "the biggest must", as many of us are waiting for Vive support ;P anyhow its the Author's decision on what to do next.

**Comment by budwheizzah** [comment]
Funny thing... implementing VIVE support would indirectly address my biggest "must" ;)

**Comment by cercata** [comment]
_"you should avoid playing this game with the gamepad, you'll get sick"_

I don't get sick playing with gamepad !!!!!

**Comment by Nibre** [comment]
Scale is tricky to perceive in this game, as your character’s small, space-stations are cramped, and VR skews floor perception when sitting.

From everything that I can tell, the scale is perfectly matched to the objects in the game. Headset IPD is what will greatly affect your perception of scale, so make sure that you have it set correctly, and keep in mind spaceships are supposed to feel cramped.

**Comment by Nibre** [comment]
Also, Vive support was just added in this latest release ;-).

**Comment by budwheizzah** [comment]
Wow! VIVE?! Awesome!

Ok, on the scale I have to respectfully disagree. Again, the most telling element is any human model (the androids, the people, etc) which clearly appear not massive enough (in terms of 2D everything looks fine, but the 3D depth is so damn strong, the people look like 4 foot tall dolls, and I feel like a giant head moving with half my body just protruding below the floor). Yes spaceships should feel cramped, but it should actually fit the size of a human being which right now it doesn't.

IPD has never changed the appearance of scale in this game; believe me I've tried. With the DK2 this scale issue was much, much worse (if you need a reference, do feel free to try it stock with a DK2 you'll see what I mean). What would be nice is a setting to adjust the stereoscopic camera's separation strength (a bit like VorpX offers the option). This way we can either leave it be, or adjust it if wanted. I'm sure you can tap into that since you've exposed a lot of settings so far.

I do scale evaluations time and time again, because I make VorpX config tutorials on my channel... I hope you can trust when I say, I know an incorrect scale when I see one, and this is definitely 105% certainly wrong.

**Comment by cercata** [comment]
For me the scale is OK ... maybe it's only ok for people with average IPD

**Comment by baggyg** [comment]
The scale also feels fine for me. I have a very narrow 58mm IPD. I also spent a lot of time with World Scale whilst working on Vireio Perception. If anything you are slightly too tall as passing through a doorway is a bit close (although I haven't played with trying to reset my height) . It would be great if there was a world scale option to experiment, however I presume if this is not accessible the only option would be some sort of matrix modification, which is presumably a lot of work.
